pkgs/applications/graphics/inkscape/extensions/inkstitch/default.nix

```
@@ -5,22 +5,45 @@
   gettext,
 }:
 let
-  version = "3.1.0";
+  # on update check compatibility to nixpkgs inkex
+  version = "3.2.2";
+
+  # remove and use nixpkgs version is recent enough
+  inkex-updated = python3.pkgs.inkex.overrideAttrs (old: rec {
+    # this is no special commit, just the most recent as of writing
+    version = "3150a5b3b06f7e4c2104d9e8eb6dc448982bb2b0";
+    src = fetchGit {
+      url = "https://gitlab.com/inkscape/extensions.git";
+      rev = "${version}";
+      allRefs = true;
+    };
+
+    # (not needed for this version)
+    doCheck = false;
+
+    postPatch = ''
+      substituteInPlace pyproject.toml \
+        --replace-fail 'scour = "^0.37"' 'scour = ">=0.37"'
+    '';
+  });
+
   dependencies =
     with python3.pkgs;
     [
       pyembroidery
-      inkex
+      # inkex upstream release & nixpkgs is too old
+      # use nixpkgs inkex once up to date
+      inkex-updated
```

```suggestion
      # inkex upstream release & nixpkgs is too old
      # use nixpkgs inkex once up to date
      inkex'
```
or we update that to a prerelease?